core/src/OpenACC/Kokkos_OpenACC_ParallelScan_Range.hpp

```diff
@@ -201,6 +202,164 @@ class ParallelScanOpenACCBase {
                               final_reducer)async(async_arg)
     acc_wait(async_arg);
   }
+#else
+  // Alternative implementation to work around OpenACC features not yet
+  // implemented by Clacc
```

Comment:
What's the difference between the two implementations?

Reply:
The current implementation of Clacc does not support gang-private variables; thus, the alternative implementation allocates the gang-private arrays on GPU global memory using array expansion.

Reply:
Updated the comment.